WalletWasabi.Fluent/Helpers/ApplicationHelper.cs

```diff
@@ -21,4 +21,25 @@ public static IObservable<string> ClipboardTextChanged(IScheduler? scheduler = d
 					.WhereNotNull())
 			.DistinctUntilChanged();
 	}
+
+	public static IObservable<bool> MainWindowActivated
+	{
+		get
+		{
+			if (MainWindow is not { } mainWindow)
+			{
+				return Observable.Return(true);
```

Comment:
If there is no main window I think it should return `false` as a nonexisting window cannot be active.

Reply:
That's true, but I still would like to know how this will impact platforms when there's no MainWindow.